### ISSUE TITLE
Unexpected token *

### ISSUE THREAD

**Issue by AleCaste** [nexe/nexe#198]
After nexe finishes compiling ok, I get this error when running the executable:
`return co(function *() {`
I guess is due to the fact that the nodearg "--harmony-generators" needs to be used.
Where do I include this flag on nexe settings?
Further than that we need to specify some default args to be passed to the final executable that is generated. Is it possible?
And finally, is it possible to include a WHOLE folder in the 'resourceFiles' setting?

Thanks,
Alex

**Comment by jaredallard** [comment]
Hi! This is covered in the `README.md`, but i'll reiterate it.

In your package.json:

```js
....
    "runtime": {
        "framework": "node",
        "version": "5.6.0",
        "js-flags": "--use_strict",
        "ignoreFlags": true
    }
}
```

**Comment by AleCaste** [comment]
Thanks, I added the flags there and the process went ok; the error got away.
... but now I get "Extension or internal compilation error." after running the executable file produced by nexe. 
There are no further hints so my guess is that our project could be using some native modules.... could this be the cause?
During the compilation process we got several warnings like the following:
```D:\Program Files (x86)\Visual Studio\VC\include\xlocale(337): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc [D:\proxy\temp\nexe\node\0.12.5\node-v0.12.5\deps\v8\tools\gyp\v8_base.vcxproj]```
